src/TokenHopper.sol

```diff
@@ -84,13 +90,10 @@ contract TokenHopper is ITokenHopper, Ownable {
      *
      * Subsequent calls to load() after the initial call will revert.
      *
-     * This function will pull in initialAmount of the token, so the caller
-     * must have properly set their allowances.
-     *
      * @param config the Hopper Configuration defining the behavior 
      */
     function load(HopperConfiguration calldata config) onlyOwner external {
```

Comment:
why isn't this just part of the constructor?

Reply:
moved to constructor as part of https://github.com/Layr-Labs/EigenHopper/pull/2/commits/eb9a87f9beee2ad45ea1b4d95b57236487b1b88e -- I think it's a good simplification